zeebe-workflow-manager/src/main/resources/bpmn/money-loan.bpmn

```diff
@@ -0,0 +1,149 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<bpmn:definitions xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" xmlns:zeebe="http://camunda.org/schema/zeebe/1.0" xmlns:modeler="http://camunda.org/schema/modeler/1.0" id="Definitions_0hmvvj0" targetNamespace="http://bpmn.io/schema/bpmn" exporter="Camunda Modeler" exporterVersion="5.13.0" modeler:executionPlatform="Camunda Cloud" modeler:executionPlatformVersion="8.2.0">
+  <bpmn:process id="money-loan" isExecutable="true">
+    <bpmn:startEvent id="StartEvent_1" name="Needed to loan money">
+      <bpmn:outgoing>Flow_03ei1b4</bpmn:outgoing>
+    </bpmn:startEvent>
+    <bpmn:exclusiveGateway id="Gateway_0wveo0b" name="Check credit" default="Flow_0kwk6ch">
+      <bpmn:incoming>Flow_03ei1b4</bpmn:incoming>
+      <bpmn:outgoing>Flow_0kwk6ch</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1sw3gpt</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:endEvent id="Event_19fj3y9" name="End">
+      <bpmn:incoming>Flow_1e0z4vc</bpmn:incoming>
+      <bpmn:incoming>Flow_02lkdv4</bpmn:incoming>
+    </bpmn:endEvent>
+    <bpmn:serviceTask id="send-acceptance-letter" name="Send acceptance letter">
+      <bpmn:extensionElements>
+        <zeebe:taskDefinition type="send-acceptance-letter" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_0kwk6ch</bpmn:incoming>
+      <bpmn:outgoing>Flow_1btw3rn</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="transfer-money" name="Transfer money">
+      <bpmn:extensionElements>
+        <zeebe:taskDefinition type="transfer-money" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1btw3rn</bpmn:incoming>
+      <bpmn:outgoing>Flow_1e0z4vc</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:serviceTask id="send-rejection-letter" name="Send rejection letter">
+      <bpmn:extensionElements>
+        <zeebe:taskDefinition type="send-rejection-letter" />
+      </bpmn:extensionElements>
+      <bpmn:incoming>Flow_1sw3gpt</bpmn:incoming>
+      <bpmn:incoming>Flow_116u7bb</bpmn:incoming>
+      <bpmn:outgoing>Flow_1td2sw1</bpmn:outgoing>
+    </bpmn:serviceTask>
+    <bpmn:exclusiveGateway id="Gateway_07jlc9k" name="Customer receive letter?" default="Flow_1m61898">
+      <bpmn:incoming>Flow_1td2sw1</bpmn:incoming>
+      <bpmn:outgoing>Flow_02lkdv4</bpmn:outgoing>
+      <bpmn:outgoing>Flow_1m61898</bpmn:outgoing>
+    </bpmn:exclusiveGateway>
+    <bpmn:intermediateCatchEvent id="Event_09vew4m" name="Wait 30 seconds">
+      <bpmn:incoming>Flow_1m61898</bpmn:incoming>
+      <bpmn:outgoing>Flow_116u7bb</bpmn:outgoing>
+      <bpmn:timerEventDefinition id="TimerEventDefinition_1k2y3cd">
+        <bpmn:timeDuration xsi:type="bpmn:tFormalExpression">PT30S</bpmn:timeDuration>
```

Comment:
Still bit short imo, to debug the timer functionalities in the UI

Reply:
@jaakkoraut update to PT120S (2 minutes)